tools/pythonpkg/tests/fast/test_pyarrow_udf.py

```diff
@@ -0,0 +1,209 @@
+import duckdb
+import os
+import pytest
+pd = pytest.importorskip("pandas")
+pa = pytest.importorskip("pyarrow")
+from typing import Union
+import pyarrow.compute as pc
+
+from duckdb.typing import *
+
+class TestPyArrowUDF(object):
+
+    def test_basic_use(self):
```

Comment:
Can we add a function that tests all supported types? And tests with not supported types?

Reply:
Also, tests, where you validate > vector size data (I've seen some length checks, but did not find the actual data), a mix of nulls and non-nulls